browser/main/modals/PreferencesModal/BackupTab.styl

```diff
@@ -0,0 +1,160 @@
+@import('./ConfigTab')
+
+.backup
+  margin-bottom 15px
+
+.backup-header
+  font-size 24px
+  color $ui-text-color
+  padding 5px
+  border-bottom $default-border
+  margin-bottom 15px
+
+.backup-body-section
+  margin-bottom 15px
+  display flex
+  line-height 30px
+
+.backup-body-section-label
+  width 150px
+  margin-right 60px
+  margin-top 5px
+
+.backup-body-section-name
+  flex 1
+.backup-body-section-name-input
+  height 30px
+  vertical-align middle
+  width 150px
+  font-size 12px
+  border solid 1px $border-color
+  border-radius 2px
+  padding 0 5px
+
+.backup-body-section-path
+  flex 1
+.backup-body-section-path-input
+  height 30px
+  vertical-align middle
+  width 150px
+  font-size 12px
+  border-style solid
+  border-width 1px 0 1px 1px
+  border-color $border-color
+  border-top-left-radius 2px
+  border-bottom-left-radius 2px
+  padding 0 5px
+  margin-top 4px
+.backup-body-section-path-button
+  margin-top 4px
+  height 30px
+  border none
+  border-top-right-radius 2px
+  border-bottom-right-radius 2px
+  colorPrimaryButton()
+  vertical-align middle
+.backup-body-control
+  border-top $default-border
+  padding-top 10px
+  box-sizing border-box
+  height 40px
+  text-align right
+
+.backup-button-control
+  position relative
+
+.backup-submit-leftButton
+  colorPrimaryButton()
+  border none
+  border-radius 2px
+  font-size $tab--button-font-size
+  height $tab--button-height
+  padding 0 15px
+  margin-left 25px
+
+body[data-theme="dark"]
+  .root
+    padding 15px
+    color $ui-dark-text-color
+
+  .backup-header
+    color $ui-dark-text-color
+    border-color $ui-dark-borderColor
+
+  .backup-body-section-name-input
+    border-color $ui-dark-borderColor
+
+  .backup-body-section-path-button
+    colorPrimaryButton()
+      
+  .backup-body-control
+    border-color $ui-dark-borderColor
+
+  .backup-submit-leftButton
+    colorDarkDefaultButton()
+    border-color $ui-dark-borderColor
+
+
+
+body[data-theme="solarized-dark"]
+  .root
+    color $ui-solarized-dark-text-color
+
+  .backup-header
+    color $ui-solarized-dark-text-color
+    border-color $ui-solarized-dark-borderColor
+
+  .backup-body-section-name-input
+    border-color $$ui-solarized-dark-borderColor
+
+  .backup-body-section-path-button
+    colorPrimaryButton()
+
+  .backup-body-control
+    border-color $ui-solarized-dark-borderColor
+
+  .backup-submit-leftButton
+    colorDarkDefaultButton()
+    border-color $ui-solarized-dark-borderColor
+
+body[data-theme="monokai"]
```

Comment:
We have recently changed the way we style the component so that we can reduce the amount of code written. Can you change it to the way we currently styling now? An example can be seen here:

This is the old way of styling:
https://github.com/BoostIO/Boostnote/blob/8b8d915ab7dff856dc64e637609498fae9bd6225/browser/styles/Detail/TagSelect.styl

This is the new way of styling:
https://github.com/BoostIO/Boostnote/blob/master/browser/styles/Detail/TagSelect.styl